main.go

```diff
@@ -22,51 +22,74 @@ func main() {
 
 	// Iterate over each pull request
 	for _, pr := range prs {
-		fmt.Printf("PR #%d: %s\n", pr.GetNumber(), pr.GetTitle())
-
-		fmt.Printf("Getting diff for PR #%d\n", pr.GetNumber())
-		diff, err := client.GetPrDiff(pr)
-		if err != nil {
-			log.Fatalf("Failed to get raw diff: %v", err)
+		comments := reviewPR(pr, client)
+		commitId := pr.GetHead().GetSHA()
+		for _, comment := range comments {
+			// post to github
+			fmt.Printf("Posting comment to PR #%d\n", pr.GetNumber())
+			ghComment := client.ParsedCommentToGithubComment(comment, commitId)
```

Comment:
Bug: there is no check for errors when posting a comment to GitHub, which could result in lost comments and failed reviews

Reply:
see the next line